NEWS.md

```diff
@@ -1,3 +1,18 @@
+flux-accounting version 0.4.0 - 2020-11-06
+------------------------------------------
+
+#### Fixes
+
+* `view-job-records` subcommand parameters adjusted to be unpacked as a dictionary (#55)
+
+* Move `view_job_records()` and its helper functions into its own module (#57)
```

Comment:
```suggestion
* Move `view_job_records()` and its helper functions into its own python module (#57)
```

Flux heavily uses the term module like flux service module ... Would be good to say this is a "python" module if this really is the case.